crates/edr_napi/src/chains/op.rs

```diff
@@ -208,6 +208,11 @@ pub fn op_genesis_state(hardfork: OpHardfork) -> Vec<AccountOverride> {
                 // uint256 blobBaseFee = 10 gwei
                 value: BigInt::from(0x00000002540be400_u64),
             },
+            StorageSlot {
+                // Operator fee parameters
+                index: BigInt::from(8u64),
+                value: BigInt::from(0u64),
+            },
```

Comment:
Issue: Since this storage value was not present before the Isthmus hardfork, the `AccountOverride` should be different depending on the hardfork too.